src/lib/dependencies.ts

```diff
@@ -65,9 +75,14 @@ export function resolveAllDependencies (options: Options): Promise<DependencyTre
  */
 export function resolveDependency (dependency: Dependency, options: Options): Promise<DependencyTree> {
   const { type, location, raw, meta } = dependency
-
   if (type === 'registry') {
+    // DEBUGGING
+    console.log('resolveDependency starts', location, raw)
```

Comment:
Shouldn't be. HTTP calls are throttled from within the codebase though, I think to around 10 at a time.